qadence/ml_tools/utils.py

```diff
@@ -1,16 +1,21 @@
 from __future__ import annotations
 
+import os
+from dataclasses import dataclass
 from functools import singledispatch
 from typing import Any
 
 from torch import Tensor, rand
 
 from qadence.blocks import AbstractBlock, parameters
 from qadence.circuit import QuantumCircuit
+from qadence.logger import get_script_logger
 from qadence.ml_tools.models import TransformedModule
 from qadence.models import QNN, QuantumModel
 from qadence.parameters import Parameter, stringify
 
+logger = get_script_logger(__name__)
+
```

Comment:
```suggestion
from typing import Any
from logging import get_logger

from torch import Tensor, rand

from qadence.blocks import AbstractBlock, parameters
from qadence.circuit import QuantumCircuit
from qadence.ml_tools.models import TransformedModule
from qadence.models import QNN, QuantumModel
from qadence.parameters import Parameter, stringify

logger = get_logger(__name__)

```

Reply:
why? dont you wanna use your creation :)

Reply:
well, it's part of core qadence, so it can use the standard logging mechanism